package.json

```diff
@@ -16,6 +16,7 @@
     "coverage": "nyc report",
     "coverage:html": "nyc report --reporter=lcov && open-cli coverage/lcov-report/index.html",
     "build": "rollup --config ./build/rollup.config.js",
+    "build:watch": "rollup --config ./build/rollup.config.js --watch",
```

Comment:
Do not forget to update `README.md` with the new command.

Reply:
You are right.
Readme updated